modules/loader/src/main/java/com/opengamma/strata/loader/LoaderUtils.java

```diff
@@ -265,6 +292,17 @@ private static String normalizeIfBracketed(String value) {
     }
   }
 
+  private static String normalizeIfCommaSeparated(String value) {
+    if (!value.startsWith(",") && !value.startsWith("-,") && !value.endsWith(",") && !value.contains(",,")) {
+      String[] parts = value.split("\\.");
```

Comment:
Setup a private static constant for `Splitter.on('.')` and use it here to avoid creating a regex each time